# Importation

In [2]:
#importation des packages

import pandas as pd
import matplotlib.pyplot as plt
import seaborn
import random

In [3]:
#importation du jeu de données
data = pd.read_csv("annual_deaths_by_causes.csv", delimiter = ",")
df = data.copy()

# Analyse exploratoire

In [4]:
df.shape

(7273, 35)

In [5]:
df.head(100)

country code  year  meningitis  alzheimer's_diesease  \
0   Afghanistan  AFG  2007      2933.0                1402.0   
1   Afghanistan  AFG  2008      2731.0                1424.0   
2   Afghanistan  AFG  2009      2460.0                1449.0   
3   Afghanistan  AFG  2011      2327.0                1508.0   
4   Afghanistan  AFG  2012      2254.0                1544.0   
..          ...  ...   ...         ...                   ...   
95      Algeria  DZA  1995       678.0                1763.0   
96      Algeria  DZA  1996       650.0                1855.0   
97      Algeria  DZA  1997       637.0                1945.0   
98      Algeria  DZA  1998       622.0                2037.0   
99      Algeria  DZA  1999       602.0                2129.0   

    parkinson's_disease  nutritional_deficiency  malaria  drowning  \
0                 450.0                  2488.0    393.0    2127.0   
1                 455.0                  2277.0    255.0    1973.0   
2                 460.0                  2040.0    239.0    1852.0   
3                 473.0                  1846.0    390.0    1775.0   
4                 482.0                  1705.0     94.0    1716.0   
..                  ...                     ...      ...       ...   
95                525.0                   385.0      7.0    1061.0   
96                540.0                   367.0      5.0    1035.0   
97                554.0                   356.0      5.0    1042.0   
98                565.0                   342.0      2.0    1016.0   
99                582.0                   324.0      1.0    1001.0   

    interpersonal_violence  ...  chronic_kidney_disease  poisonings  \
0                   3657.0  ...                  4490.0       512.0   
1                   3785.0  ...                  4534.0       495.0   
2                   3874.0  ...                  4597.0       483.0   
3                   4170.0  ...                  4785.0       483.0   
4                   4245.0  ...                  4846.0       482.0   
..                     ...  ...                     ...         ...   
95                   600.0  ...                  3856.0       456.0   
96                   559.0  ...                  3906.0       452.0   
97                   831.0  ...                  3937.0       455.0   
98                   736.0  ...                  3962.0       451.0   
99                   616.0  ...                  4060.0       452.0   

    protein_energy_malnutrition  terrorism  road_injuries  \
0                        2439.0     1199.0         7425.0   
1                        2231.0     1092.0         7355.0   
2                        1998.0     1065.0         7290.0   
3                        1805.0     1525.0         7432.0   
4                        1667.0     3521.0         7494.0   
..                          ...        ...            ...   
95                        354.0      418.0        13374.0   
96                        336.0      529.0        13410.0   
97                        326.0     4266.0        13558.0   
98                        313.0      952.0        13525.0   
99                        295.0      467.0        13631.0   

    chronic_respiratory_diseases  chronic_liver_diseases  digestive_diseases  \
0                         7222.0                  3346.0              6458.0   
1                         7143.0                  3316.0              6408.0   
2                         7045.0                  3291.0              6358.0   
3                         6916.0                  3318.0              6370.0   
4                         6878.0                  3353.0              6398.0   
..                           ...                     ...                 ...   
95                        4846.0                  2632.0              4242.0   
96                        4903.0                  2670.0              4281.0   
97                        4960.0                  2718.0              4339.0   
98          

In [8]:
df.dtypes.value_counts()

float64    32
object      2
int64       1
dtype: int64

# Contexte statistique
- Population : 7273 ensembles de décès annuels 
- Nature des variables : 33 variables quantitatives discrètes  et 2 variables qualitatives nominales